app/controllers/flashcard_templates_controller.rb

```diff
@@ -0,0 +1,61 @@
+class FlashcardTemplatesController < ApplicationController
+  before_action :find_flashcard_template, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @flashcard_templates = policy_scope(FlashcardTemplate)
+    @flashcard_template = FlashcardTemplate.new
+    authorize @flashcard_template
+  end
+
+  def new
+    @flashcard_template = FlashcardTemplate.new
+    authorize @flashcard_template
+    @topics = Topic.all.plug
+  end
+
+  def create
+    @flashcard_template = FlashcardTemplate.new(flashcard_template_params)
+    @flashcard_template.teacher = current_teacher
+    authorize @flashcard_template
+    @flashcard_set = FlashcardSet.find(params[:flashcard_set_id])
+    @flashcard_template.flashcard_set = @flashcard_set
```

Comment:
I'd just put the authorize after asigning the set

Reply:
Why?